src/microbe_stage/MicrobeStage.cs

```diff
@@ -134,6 +139,12 @@ public List<Node> SavedGameEntities
         set => savedGameEntities = value;
     }
 
+    /// <summary>
+    ///   True once stage fade-in is complete
+    /// </summary>
+    [JsonIgnore]
+    public bool TransitionFinished { get; internal set; }
```

Comment:
Is this functionally the same as { get; set; }?

Reply:
Basically yeah. If we had multiple modules, only the code in the same module can call internal stuff. So I used that here to mark that not everyone should use the setter.